### PR TITLE
Fix #6825 by reducing the changed files to those only related to the issue

### LINKED ISSUE #3275: Exporting bindings not working
**OlaK** created:
https://orchard.codeplex.com/workitem/19447

I'm having problems exporting (via the Export module) a binding I've created through the Bindings tab on the Query page, has anyone come across this before? 

I tried first on my project I'm working on, when that didn't work, I set up a clean install of Orchard 1.6 and tried there. However the binding does not appear in the exported xml, the query I created pointing at the binding comes out ok and is still pointing to the type. 

I checked in the error log but there's nothing in there... 

I'm attaching the exported xml, see the query with title "Test binding Author Export Query".

### PR DESCRIPTION
**Pull request by HengzheLi**
#6825 fixed #3275 by @jersiovic, but it was prevented from merging as it changed too many files.

This PR, I branch it from master, only apply the commit 106ff07 and bd531cf from jersiovic in #6825 of which were key commits for fixing #3275 and there are any other changes. 

PS: hope this PR does not offense your contributions @jersiovic as it reuse your works. If it does, I will close it.

### CODE REVIEW

**Comment by jersiovic** [comment]
@li0803 no problem,  thank you for submitting the PR

File: ClickToBuild.cmd [hunk 1]
```diff
@@ -1,15 +1,26 @@
-for /f "usebackq tokens=*" %%i in (`lib\vswhere\vswhere -latest -version "[15.0,16.0)" -requires Microsoft.Component.MSBuild -property installationPath`) do (
```
Comment (jersiovic) [inline ClickToBuild.cmd:1]: Changes in this file are really needed? It looks that are not really related with thi topic of the PR

File: src/Orchard.Web/Modules/Orchard.Projections/Orchard.Projections.csproj [hunk 2]
```diff
@@ -168,6 +168,8 @@
     <Compile Include="Models\NavigationQueryPart.cs" />
     <Compile Include="Models\NavigationQueryPartRecord.cs" />
     <Compile Include="Models\QueryPart.cs" />
+    <Compile Include="Providers\Builders\MemberBindingsStep.cs" />
+    <Compile Include="Providers\Excutors\MemberBindingsStep.cs" />
```
Comment (jersiovic) [inline src/Orchard.Web/Modules/Orchard.Projections/Orchard.Projections.csproj]: Typo Excutors should be Executors

File: src/Orchard.Web/Modules/Orchard.Projections/Providers/Builders/MemberBindingsStep.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using Orchard.Data;
+using Orchard.Localization;
+using Orchard.Projections.Services;
+using Orchard.Recipes.Services;
+
+namespace Orchard.Projections.Providers.Builders
+{
```
Comment (jersiovic) [inline src/Orchard.Web/Modules/Orchard.Projections/Providers/Builders/MemberBindingsStep.cs]: Better use curly braces on the same line following style used in Orchard for consistency for all the files in this PR

**Review by jersiovic — CHANGES_REQUESTED** [review]

File: ClickToBuild.cmd (hunk 1 above)
Comment (jersiovic) [reply ClickToBuild.cmd:1]: Well, the topic of the PR in reality is very open

**Comment by sebastienros** [comment]
Thanks, please apply the changes requested by @jersiovic

**Comment by HengzheLi** [comment]
apply jersiovic 2) and 3) review suggestion

File: ClickToBuild.cmd (hunk 1 above)
Comment (HengzheLi) [reply ClickToBuild.cmd:1]: I made this branch from master. And this change is a head commit to 1.10.x by sebastienros on master branch. 
And master is a stable branch. So I think I'd better submit this PR to another one.
